### DISCUSSION TITLE
Laser engraving

### DISCUSSION THREAD

**Discussion by rmbom** [fooandbar/Mks-Robin-Nano-Marlin2.0-Firmware-For-Selpic-Star-A.LABISTS-SX1#10] [General]
Hello there!

I was wondering if you were able to make any progress on the laser engraving part of the firmware? Is there any way we can help out on this? I've pretty much given up on getting any usable 3D prints from this thing, but I did see some people getting good results from the laser. Sadly I did see the notice about not being able to go back to the original firmware after installing this one, but did not realise this meant not being able to use the laser.

Thank you for all your time and hard work on this!

**Comment by fooandbar** [comment]
Hi,
Current version enables laser engraving option.
<img src='https://user-images.githubusercontent.com/63621440/123628366-f89c3400-d84d-11eb-9fab-29449d376b14.png' width=300 />
But this version has no support of `S` parameter in gcode motion mode, so I need some tips to print.
